plugin_tests/dataone_test.py

```diff
@@ -10,6 +10,7 @@
 from girder.constants import ROOT_DIR
 from girder.api.rest import RestException
 
+base.TestCase.maxDiff = None
```

Comment:
is this needed? https://docs.python.org/3/library/unittest.html#unittest.TestCase.maxDiff